elements/pfe-tabs/src/pfe-tab-panel.scss

```diff
@@ -3,21 +3,29 @@
 $LOCAL: tabs;
 
 $LOCAL-VARIABLES: (
-  focus:  pfe-var(link)
+
+  focus:  pfe-var(link),
+  panel: ( // to avoid collisions with tab header
+      BackgroundColor: transparent,
+      Padding:         pfe-var(container-spacer),
+  )
 );
+// DEBUGGING
+//@include pfe-local-debug; // preview local variables in pfe-tab-panel.css
 
-:host {
+////////////////////////////////
 
+:host {
   display: block;
+  color: pfe-broadcasted(text);
 
   @at-root #{&}(:focus) {
     outline: none;
   }
 
   ::slotted(*) {
```

Comment:
If there are no other styles in here, you can delete the slotted selector

Reply:
+1, updated